src/Features/CSharp/Portable/InitializeParameter/CSharpAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -49,5 +56,14 @@ protected override bool CanOffer(SyntaxNode body)
 
             return true;
         }
+
+        protected override SyntaxNode GetParameterNodeAtIndex(int ordinal, SyntaxNode functionDecleration)
+        {
+            var listOfParameters = InitializeParameterHelpers.GetParameters(functionDecleration);
+
+            return listOfParameters.Length < ordinal
```

Comment:
This would throw if `listOfParameters` is default. 
See [http://index/?leftProject=System.Collections.Immutable&file=System%5CCollections%5CImmutable%5CImmutableArray_1.Minimal.cs&line=163](http://index/?leftProject=System.Collections.Immutable&file=System%5CCollections%5CImmutable%5CImmutableArray_1.Minimal.cs&line=163)